### PR TITLE
editingandcorrectingshowbysarakhaled

### PR DESCRIPTION
**Pull request by engsarakhaled**
i make correction for show route,controller,blades and edit_job

### CODE REVIEW

File: app/Http/Controllers/PublicController.php [hunk 1]
```diff
@@ -28,28 +30,16 @@ public function category()
...
-    
-    public function jobdetails(String $id)
-    {
-        $job=JobData::with(['company'])->findOrFail($id);
-        dd($job);
-        return view('public.job-detail',compact('job'));
-    }
-
-    public function jobApply(Request $request)
+    public function jobdetails()
```
Comment (SafaaMagdyAwad) [inline app/Http/Controllers/PublicController.php:40]: I can't understand the reason for this change.

File: resources/views/public/includes/job-details.blade.php
Comment (SafaaMagdyAwad) [inline resources/views/public/includes/job-details.blade.php:1]: Is this page static?  Why

File: app/Http/Controllers/PublicController.php (hunk 1 above)
Comment (engsarakhaled) [reply app/Http/Controllers/PublicController.php:40]: You asks for the changes in testimonials??

**Comment by SafaaMagdyAwad** [comment]
No im asking about removing code in joblist

File: app/Http/Controllers/PublicController.php (hunk 1 above)
Comment (SafaaMagdyAwad) [reply app/Http/Controllers/PublicController.php:40]: The code in jobdetails and in jobaApply  are removed.

Comment (engsarakhaled) [reply app/Http/Controllers/PublicController.php:40]: I didn't remove it .esraa remove it in the last update .and we are working on admin files not public this code was in the public as I remember and ur code was editing and show in public page and this isn't correct

**Pull request merged**